packages/@aws-cdk/aws-ecs-patterns/lib/base/application-load-balanced-service-base.ts

```diff
@@ -351,21 +351,23 @@ export abstract class ApplicationLoadBalancedServiceBase extends cdk.Construct {
         throw new Error('A Route53 hosted domain zone name is required to configure the specified domain name');
       }
 
-      const record = new ARecord(this, "DNS", {
-        zone: props.domainZone,
-        recordName: props.domainName,
-        target: RecordTarget.fromAlias(new LoadBalancerTarget(loadBalancer)),
-      });
+      if (internetFacing) {
+        const record = new ARecord(this, "DNS", {
+          zone: props.domainZone,
+          recordName: props.domainName,
+          target: RecordTarget.fromAlias(new LoadBalancerTarget(loadBalancer)),
+        });
 
-      domainName = record.domainName;
+        domainName = record.domainName;
+      }
     }
 
+    new cdk.CfnOutput(this, 'LoadBalancerDNS', { value: loadBalancer.loadBalancerDnsName });
```

Comment:
Just out of curiosity: why making this change?

Reply:
oops I guess this is not necessary. Will move these 2 lines back to where they were